File: java/Kode program/PowerTypeInfo.java
public class PowerTypeInfo {
    private String sourceType;
    private int wattage;

    public void setSourceType(String st) { sourceType = st; }
    public String getSourceType() { return sourceType; }

    public void setWattage(int w) { wattage = w; }
    public int getWattage() { return wattage; }

    public void printPowerSource() {
        System.out.println("--- Power Type Info ---");
        System.out.println("Source Type: " + sourceType);
        System.out.println("Wattage: " + wattage + " W");
    }
}

File: python/Kode program/PowerTypeInfo.py
class PowerTypeInfo:
    def __init__(self):
        self.sourceType = ""
        self.wattage = 0

    def setSourceType(self, st):
        self.sourceType = st

    def getSourceType(self):
        return self.sourceType

    def setWattage(self, w):
        self.wattage = w

    def getWattage(self):
        return self.wattage

    def printPowerSource(self):
        print("--- Power Type Info ---")
        print(f"Source Type: {self.sourceType}")
        print(f"Wattage: {self.wattage} W")
